include/swift/ABI/MetadataValues.h

```diff
@@ -735,6 +733,13 @@ enum class FunctionMetadataConvention: uint8_t {
   CFunctionPointer = 3,
 };
 
+/// Differentiability kind for function type metadata.
+enum class DifferentiabilityKind: uint8_t {
```

Comment:
This should not be moved, but should be duplicated instead. Similarly like how `TargetFunctionTypeFlags` duplicates `ExtInfo`.

Reply:
How can we duplicate?
Currently, both are defined in the `swift` namespace as `swift::DifferentiabilityKind`.

Reply:
Created `FunctionMetadataDifferentiabilityKind` in https://github.com/apple/swift/pull/25804/commits/3c1f0218dd14a87a54db31d58d97a2c5411e7297, mimicking the existing `FunctionMetadataConvention`.